backend/src/docs/v1.api-spec.yaml

```diff
@@ -268,6 +270,12 @@ components:
             type: string
             example: CMSG
           description: An array of applicationAcronyms
+        passwordPublicKey:
+          type: string
+          example: MIGfMA0GCSqGS...
+        webadeEnvironment:
+          type: string
+          example: INT
```

Comment:
A potential description saying one of `INT`, `TEST` or `PROD` may be useful here - helps with API expectations.